.github/workflows/docker-image.yml

```diff
@@ -6,6 +6,10 @@ on:
       - main
       - dev
       - staging
+  pull_request:
```

Comment:
do we want to be pushing images whenever there is a PR? seems like something that would make sense to do only when a push is made to the branch. maybe i'm over looking something